src/App/psr.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App;
+
+/**
+ * File doc
+ * PHP version 7.4
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+
+/**
+ * Class doc
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+class Psr {
```

Comment:
:no_entry_sign: **Error**: Opening brace of a class must be on the line after the definition (*PSR2.Classes.ClassDeclaration.OpenBraceNewLine*).